invoices/invoiceregistry_test.go

```diff
@@ -1369,14 +1369,14 @@ func TestSpontaneousAmpPayment(t *testing.T) {
 
 // testSpontaneousAmpPayment runs a specific spontaneous AMP test case.
 func testSpontaneousAmpPayment(
-	t *testing.T, keySendEnabled, failReconstruction bool, numShards int) {
+	t *testing.T, ampEnabled, failReconstruction bool, numShards int) {
 
 	defer timeout()()
 
 	ctx := newTestContext(t)
 	defer ctx.cleanup()
 
-	ctx.registry.cfg.AcceptKeySend = keySendEnabled
+	ctx.registry.cfg.AcceptAMP = ampEnabled
```

Comment:
any specific reason we want to only test this for AMP now, and not keysend? (because we are deprecating keysend?)

Reply:
before the behavior was gated under `accept-keysend` because the `accept-amp` flag didn't exist, this just converts the test to use the proper config value